cmd/limitometer/main.go

```diff
@@ -74,9 +75,24 @@ func main() {
 	if exists {
 		*nodename = env
 	}
+	confval, exists := os.LookupEnv("METADATA")
```

Comment:
Can we name this `LIMITOMETER_CONFIG` maybe?

Reply:
Updated value to  LIMITOMETER_CONFIG